FinanceTracker/FinanceTracker/Commands/GetStatisticsBase.cs

```diff
@@ -0,0 +1,35 @@
+﻿using FinanceTracker.Enums;
+using FinanceTracker.Interfaces;
+using FinanceTracker.Models;
+
+namespace FinanceTracker.Commands
+{
+    public abstract class GetStatisticsBase : ICommand<decimal>
+    {
+        protected abstract DateTime GetStartDate();
+        public decimal Execute(Account account)
+        {
+            DateTime startDate = GetStartDate();
+            DateTime endDate = DateTime.UtcNow;
+
+            decimal total = 0.0m;
+
+            var operations = account.Operations
+                .Where(operation => operation.Date >= startDate && operation.Date <= endDate);
+
+            foreach (var operation in operations)
```

Comment:
I think you can use Aggregate here https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable.aggregate?view=net-8.0